<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RawCorrectionClass</class>
 <widget class="QMainWindow" name="RawCorrectionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RawCorrection</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="AirButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>空气校正</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeadPixelButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>坏点校正</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlatFieldButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>平场校正</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DarkFieldButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>暗场校正</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ImportProjButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>导入原图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowProjButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示原图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ImportDarkButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>导入暗场图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ImportBkgButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>导入背景图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowDarkButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示暗场图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowBkgButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示背景图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveImageButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存处理图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowImageButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示处理图</string>
    </property>
   </widget>
   <widget class="QLabel" name="PicLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="setButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="HeightLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>高度:</string>
    </property>
   </widget>
   <widget class="QLabel" name="WidthLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>宽度:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入图片宽度和高度:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="HeightSpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="WidthSpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QLabel" name="AirMulLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>增益倍数:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="DeadPixelLowSpinBox">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>170</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="DeadPixelLowLabel">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>低阈值:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="DeadPixelHeightSpinBox">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>150</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="DeadPixelHeightLabel">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>高阈值:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="AirMulSpinBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="AutoCorrectionButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>一键校正</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AutoCorrectionAllButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>0</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>一键校正所有</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveAllImageButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>保存所有处理图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreatDeadPixeButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>生成坏点图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ImportProjDirButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件夹</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>610</y>
      <width>511</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="PicNumLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="PicNumScrollBar">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RawCorrection.qrc"/>
 </resources>
 <connections/>
</ui>
